app/server/utils/credit_transfers.py

```diff
@@ -361,24 +364,63 @@ def make_blockchain_transfer(transfer_amount,
 
 
 def make_payment_transfer(transfer_amount,
-                          send_user,
-                          send_transfer_account,
-                          receive_user,
-                          receive_transfer_account,
+                          token=None,
+                          send_user=None,
+                          send_transfer_account=None,
+                          receive_user=None,
+                          receive_transfer_account=None,
                           transfer_use=None,
                           transfer_mode=None,
                           require_sender_approved=True,
                           require_recipient_approved=True,
                           require_sufficient_balance=True,
                           automatically_resolve_complete=True,
-                          uuid=None):
+                          uuid=None,
+                          is_disbursement=False,
+                          is_reclaim=False):
```

Comment:
given that disbursement and reclaim can't be both true, wondering if maybe this "type" should be a string config? not certain though.

Reply:
mhm need to think about that